joint_limits:

    swing/left/joint:
      has_position_limits: true
      min_position: -3.1416/2
      max_position: 3.1416/2
      has_velocity_limits: true
      max_velocity: 6500
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 10

    swing/right/joint:
      has_position_limits: true
      min_position: -3.1416/2
      max_position: 3.1416/2
      has_velocity_limits: true
      max_velocity: 6500
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 1

    extend/joint:
      has_position_limits: true
      min_position: -3.1416/2
      max_position: 3.1416/2
      has_velocity_limits: true
      max_velocity: 0.5
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 1
